packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -489,20 +489,20 @@ class _InkResponseState<T extends InkResponse> extends State<T> with AutomaticKe
 
   bool get highlightsExist => _highlights.values.where((InkHighlight highlight) => highlight != null).isNotEmpty;
 
+  Action _createAction() {
+    return CallbackAction(
+      ActivateAction.key,
+      onInvoke: (FocusNode node, Intent intent) {
+        _startSplash(context: node.context);
+        _handleTap(node.context);
+      },
+    );
+  }
+
   @override
   void initState() {
     super.initState();
-    _actionMap = <LocalKey, ActionFactory>{
-      ActivateAction.key: () {
-        return CallbackAction(
-          ActivateAction.key,
-          onInvoke: (FocusNode node, Intent intent) {
-            _startSplash(context: node.context);
-            _handleTap(node.context);
-          },
-        );
-      },
-    };
+    _actionMap = <LocalKey, ActionFactory>{ ActivateAction.key: _createAction };
```

Comment:
Was this a performance win, or just a style choice to pull out `createAction`? Just curious.

Reply:
This was a small performance win, since we weren't creating the closure on the fly in the `initState`.